projects/packages/my-jetpack/_inc/data/notices/use-fetching-error-notice.ts

```diff
@@ -19,6 +21,8 @@ const errorNoticeWhitelist = [
 	QUERY_PURCHASES_KEY,
 	QUERY_ACTIVATE_PRODUCT_KEY,
 	QUERY_INSTALL_PRODUCT_KEY,
+	QUERY_GET_PROTECT_DATA_KEY,
+	QUERY_GET_VIDEOPRESS_DATA_KEY,
```

Comment:
I figured it would be good to show an error notice when these requests fail. If the user is used to seeing their data on the cards and the request fails, they could be confused as to why they're not seeing it as usual